lib/msal-browser/src/interaction_client/StandardInteractionClient.ts

```diff
@@ -270,23 +270,35 @@ export abstract class StandardInteractionClient extends BaseInteractionClient {
             validatedRequest.account = account;
         }
 
-        // Check for ADAL SSO
+        // Check for ADAL/MSAL v1 SSO
         if (StringUtils.isEmpty(validatedRequest.loginHint)) {
```

Comment:
Does this need to be called if `account` is provided above?

Reply:
Good catch, I don't think so? Will check.

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/de24de1d3a135c106143a6cc9907cc1264f546e0/lib/msal-common/src/client/AuthorizationCodeClient.ts#L322-L354

Looks like we should not be adding this parameter if `account` is set. The downstream will first look for `request.account`, however, it will use the `request.loginHint` before it uses the username from the account for some reason.

I'll update this conditional to check for that, but we should maybe double check the above logic (can be addressed seperately).